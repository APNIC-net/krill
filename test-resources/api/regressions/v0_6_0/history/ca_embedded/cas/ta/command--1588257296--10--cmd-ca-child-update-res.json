{
  "actor": "krill",
  "time": "2020-04-30T14:34:56.743620Z",
  "handle": "ta",
  "version": 16,
  "sequence": 10,
  "details": {
    "child_update_resources": [
      "child",
      {
        "asn": "",
        "v4": "",
        "v6": ""
      }
    ]
  },
  "effect": {
    "events": [
      16
    ]
  }
}