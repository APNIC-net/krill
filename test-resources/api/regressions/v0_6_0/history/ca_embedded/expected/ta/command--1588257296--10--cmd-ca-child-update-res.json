{
  "command": {
    "actor": "krill",
    "time": "2020-04-30T14:34:56.743620Z",
    "handle": "ta",
    "version": 16,
    "sequence": 10,
    "details": {
      "child_update_resources": [
        "child",
        {
          "asn": "",
          "v4": "",
          "v6": ""
        }
      ]
    },
    "effect": {
      "events": [
        16
      ]
    }
  },
  "result": {
    "Events": [
      {
        "id": "ta",
        "version": 16,
        "details": {
          "child_updated_resources": [
            "child",
            {
              "asn": "",
              "v4": "",
              "v6": ""
            }
          ]
        }
      }
    ]
  }
}